ush/templates/gsiparm.anl.sh

```diff
@@ -0,0 +1,195 @@
+   gomebufr       gome        metop-b   gome_metop-b         0.0     2     0
+   atmsbufr       atms        npp       atms_npp             0.0     2     0
+   atmsbufr       atms        n20       atms_n20             0.0     2     0
+   crisbufr       cris        npp       cris_npp             0.0     2     0
+   crisfsbufr     cris-fsr    npp       cris-fsr_npp         0.0     2     0 
+   crisfsbufr     cris-fsr    n20       cris-fsr_n20         0.0     2     0 
+   abibufr        abi         g16       abi_g16              0.0     2     0
+   mlsbufr        mls30       aura      mls30_aura           0.0     0     0
+   oscatbufr      uv          null      uv                   0.0     0     0
+   prepbufr       mta_cld     null      mta_cld              1.0     0     0
+   prepbufr       gos_ctp     null      gos_ctp              1.0     0     0
+   refInGSI       rad_ref     null      rad_ref              1.0     0     0
+   lghtInGSI      lghtn       null      lghtn                1.0     0     0
+   larcInGSI      larccld     null      larccld              1.0     0     0
+::
+ &SUPEROB_RADAR
+   del_azimuth=5.,del_elev=.25,del_range=5000.,del_time=.5,elev_angle_max=5.,minnum=50,range_max=100000., l2superob_only=.false.,
+ /
+ &LAG_DATA
+ /
+ &HYBRID_ENSEMBLE
+   l_hyb_ens=${ifhyb},
+   uv_hyb_ens=.true.,
+   aniso_a_en=.false.,generate_ens=.false.,
+   n_ens=${nummem},
+   beta_s0=${beta1_inv},s_ens_h=${ens_h},s_ens_v=${ens_v},
+   regional_ensemble_option=${regional_ensemble_option},
+   pseudo_hybens = .false.,
+   grid_ratio_ens = ${grid_ratio_ens},
+   l_ens_in_diff_time=.true.,
+   ensemble_path='',
+   i_en_perts_io=${i_en_perts_io},
+   jcap_ens=574,
+   fv3sar_bg_opt=${fv3lam_bg_type},
+   readin_localization=${readin_localization},
+ /
+ &RAPIDREFRESH_CLDSURF
+   dfi_radar_latent_heat_time_period=20.0,
+   metar_impact_radius=10.0,
+   metar_impact_radius_lowCloud=4.0,
+   l_gsd_terrain_match_surfTobs=.true.,
+   l_sfcobserror_ramp_t=.true.,
+   l_sfcobserror_ramp_q=.true.,
+   l_PBL_pseudo_SurfobsT=.false.,
+   l_PBL_pseudo_SurfobsQ=.false.,
+   l_PBL_pseudo_SurfobsUV=.false.,
+   pblH_ration=0.4,
+   pps_press_incr=40.0,
+   l_gsd_limit_ocean_q=.true.,
+   l_pw_hgt_adjust=.true.,
+   l_limit_pw_innov=.true.,
+   max_innov_pct=0.1,
+   l_cleanSnow_WarmTs=.true.,
+   r_cleanSnow_WarmTs_threshold=5.0,
+   l_conserve_thetaV=.true.,
+   i_conserve_thetaV_iternum=3,
+   l_gsd_soilTQ_nudge=${ifsoilnudge},
+   l_cld_bld=.true.,
+   l_numconc=.true.,
+   l_closeobs=.true.,
+   cld_bld_hgt=1200.0,
+   build_cloud_frac_p=0.50,
+   clear_cloud_frac_p=0.10,
+   iclean_hydro_withRef_allcol=1,
+   i_use_2mQ4B=0,
+   i_use_2mT4B=0,
+   i_gsdcldanal_type=${cloudanalysistype},
+   i_gsdsfc_uselist=1,
+   i_lightpcp=1,
+   i_sfct_gross=1,
+   i_coastline=3,
+   i_gsdqc=2,
+ /
+ &CHEM
+ /
+ &NST
+ /
+ &SINGLEOB_TEST
+   maginnov=1.0,magoberr=0.8,oneob_type='t',
+   oblat=38.,oblon=279.,obpres=500.,obdattim=2020040500,
+   obhourset=0.,
+ /
+"
```

Comment:
We likely want to work toward transitioning this namelist to the same management method as the other FV3 namelists for consistency by using the python functionality to build/overwrite. That will allow us to modify any of the settings via config file for more control over experiments that may want to change these settings.

Reply:
Yes, it is nice to use Jinjia template for gsiparm.anl. But this is a first step to put gsiparm.anl into track.

Reply:
This should not be managed as a jinja template, but with YAML config files and ush/set_namelist.py.

Reply:
Yes, it should be done through set_namelist.py.